app/controllers/evaluations_controller.rb

```diff
@@ -0,0 +1,21 @@
+class EvaluationsController < ApplicationController
```

Comment:
please add before_action for checking whether current_user is organiser or not (there's helper for that in application_controller)